{{cookiecutter.repo_slug}}/README.md

```diff
@@ -42,10 +42,10 @@
 {%- endif %}
 <br />
 
-[![license](https://img.shields.io/github/license/{{cookiecutter.github_username}}/{{cookiecutter.repo_slug}}.svg?style=flat-square)](LICENSE)
+[![Project license](https://img.shields.io/github/license/{{cookiecutter.github_username}}/{{cookiecutter.repo_slug}}.svg?style=flat-square)](LICENSE)
 
-[![PRs welcome](https://img.shields.io/badge/PRs-welcome-ff69b4.svg?style=flat-square)](https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.repo_slug}}/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
-[![code with hearth by {{cookiecutter.github_username}}](https://img.shields.io/badge/%3C%2F%3E%20with%20%E2%99%A5%20by-{{cookiecutter.github_username}}-ff1414.svg?style=flat-square)](https://github.com/{{cookiecutter.github_username}})
+[![Pull Requests welcome](https://img.shields.io/badge/PRs-welcome-ff69b4.svg?style=flat-square)](https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.repo_slug}}/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+[![code with love by {{cookiecutter.github_username}}](https://img.shields.io/badge/%3C%2F%3E%20with%20%E2%99%A5%20by-{{cookiecutter.github_username}}-ff1414.svg?style=flat-square)](https://github.com/{{cookiecutter.github_username}})
 
```

Comment:
typo ("hearth" presumably meant "heart" but it makes more sense to read this as "love") and clarification